aixplain/model_interfaces/interfaces/aixplain_project_node.py

```diff
@@ -0,0 +1,28 @@
+__author__='aiXplain'
+
+from abc import abstractmethod
+from kserve.model import Model
+from typing import Dict
+
+class AixplainProjectNode(Model):
+    def __init__(self, name, *args, **kwargs):
+        self.name = name
+        self.ready = False
+
+    def predict(self, request: Dict[str, Dict], headers: Dict[str, str] = None) -> Dict[str, Dict]:
+        return self.run_script(request, headers)
+
+    @abstractmethod
+    def run_script(self, input: Dict[str, Dict], headers: Dict[str, str] = None) -> Dict[str, Dict]:
+        raise NotImplementedError
+
+    def load(self) -> bool:
```

Comment:
let's call this in the init, if not it's on the user to call this and they may forget